docs/content/manual/virtual-machines/5264-edit-vm-config-after-eject-cdrom-delete-volume.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Edit vm config after Eject CDROM and delete volume
+---
+
+* Related issues: [#5264](https://github.com/harvester/harvester/issues/5264) 
+[BUG] After EjectCD from vm and edit config of vm displays empty page: "Cannot read properties of null"
```

Comment:
Seems should not new line?

Reply:
Thanks for the check. Remove the new line to concatenate them